remi/gui.py

```diff
@@ -2119,15 +2128,15 @@ class TableEditableItem(Widget, _MixinTextualWidget):
     """item widget for the TableRow."""
 
     @decorate_constructor_parameter_types([str])
-    def __init__(self, text='', **kwargs):
+    def __init__(self, text='', text_style={}, **kwargs):
```

Comment:
mutable functions args (dict in signature) are not a good idea

Reply:
Yep. If you have your hands already on this, could you change it to None and assign
text_style = text_style or {}
?